site/src/component/Review/Review.scss

```diff
@@ -26,22 +26,46 @@
   justify-self: flex-end;
   margin-right: 3vw;
 }
-.rating-label {
-  line-height: 1rem;
-  font-size: 1rem;
-}
 .rating {
   margin: 1vh 0;
   border-radius: var(--border-radius);
   width: 5vw;
   height: 5vw;
+  min-width: 7rem;
+  min-height: 7rem;
   display: flex;
   flex-direction: column;
   align-items: center;
   justify-content: center;
+  gap: 0.5rem;
+  color: white;
+}
+.rating-label {
+  line-height: 1rem;
+  font-size: 1rem;
+  font-weight: bold;
+}
+.rating-value {
   line-height: 3rem;
   font-size: 3rem;
-  color: white;
+}
+// screen width is between 1300px and 600px, inclusive
+@media only screen and (max-width: 1300px) and (min-width: 600px) {
+  .rating {
+    width: 8vw; // The rating box width & height shrink...
+    height: 8vw;
+    min-width: 5rem; // ...but only by so much.
+    min-height: 5rem;
+    gap: max(0.5vw, 0.4rem); // The same idea goes for gap...
+  }
```

Comment:
This can be a lot less confusing if we have a fixed width for the rating tiles. It's common to think that things should scale with screen size (and that was the assumption before this PR), but most of the time that's not really true.